props/prop_base.h

```diff
@@ -609,6 +637,7 @@ class PropBase : CommandParser, Looper, protected SaberBase {
     // If we're spinning the saber, require a stronger acceleration
     // to activate the clash.
     if (v > CLASH_THRESHOLD_G + fusor.gyro().len() / 200.0) {
+#endif
```

Comment:
Did you test this?
I don't think it compiles with this extra #endif in here....

Reply:
sorry, local copy had it removed, missed on github.com change, fixed now.